SDL/SDLMain.cpp

```diff
@@ -680,6 +680,9 @@ int main(int argc, char *argv[]) {
 	}
 	graphicsContext->ThreadStart();
 
+	float mouseDeltaX;
```

Comment:
These should be zero-initialized, to avoid the risk of a nonsense axis event being sent when turning on mouse control.